packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1883,6 +1989,7 @@ class EditableText extends StatefulWidget {
     properties.add(DiagnosticsProperty<bool>('enableIMEPersonalizedLearning', enableIMEPersonalizedLearning, defaultValue: true));
     properties.add(DiagnosticsProperty<bool>('enableInteractiveSelection', enableInteractiveSelection, defaultValue: true));
     properties.add(DiagnosticsProperty<SpellCheckConfiguration>('spellCheckConfiguration', spellCheckConfiguration, defaultValue: null));
+    properties.add(DiagnosticsProperty<List<String>>('contentCommitMimeTypes', contentInsertionConfiguration == null ? const <String>[] : contentInsertionConfiguration!.contentInsertionMimeTypes, defaultValue: contentInsertionConfiguration == null ? const <String>[] : kDefaultContentInsertionMimeTypes));
```

Comment:
```suggestion
    properties.add(DiagnosticsProperty<List<String>>('contentCommitMimeTypes', contentInsertionConfiguration?.contentInsertionMimeTypes ?? const <String>[]);
```

Reply:
I'll need to keep the defaultValue otherwise there's a bunch of failing tests